components/Dashboard/EntriesColumn/EntriesColumn.js

```diff
@@ -0,0 +1,37 @@
+import { Grid } from "../Dashboard.styled";
+import Link from "next/link";
+
+export default function EntriesColumn() {
+  const boxes = [
+    { gridColumnEnd: "span 4", gridRowEnd: "span 3" },
+    { gridColumnEnd: "span 4", gridRowEnd: "span 2" },
+    { gridColumnEnd: "span 1", gridRowEnd: "span 1" },
+    { gridColumnEnd: "span 1", gridRowEnd: "span 1" },
+    { gridColumnEnd: "span 1", gridRowEnd: "span 1" },
+    { gridColumnEnd: "span 1", gridRowEnd: "span 1" },
+    { gridColumnEnd: "span 2", gridRowEnd: "span 2" },
+    { gridColumnEnd: "span 2", gridRowEnd: "span 2" },
+    { gridColumnEnd: "span 2", gridRowEnd: "span 2" },
+    { gridColumnEnd: "span 2", gridRowEnd: "span 2" },
+  ];
+
+  return (
+    <Grid>
+      {boxes.map((box, index) => (
```

Comment:
you define the `boxes` yourself, so instead of having to rely on the `index` of the array, you could add an `id` to every object in your `boxes` array.